cmake/CMakeLists.txt

```diff
@@ -122,6 +124,11 @@ else()
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /d2FH4-")
     set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /d2FH4-")
   endif()
+  if(WINDOWS_STORE)
+    # cmake/external/protobuf/src/google/protobuf/compiler/subprocess.cc and onnxruntime/core/platform/windows/env.cc call a bunch of Win32 APIs.
```

Comment:
@tiagoshibata , could you please give us a guildline of which APIs can be used, which are not?

CC: @edgchen1

Reply:
Compiling with WINAPI_FAMILY set to WINAPI_FAMILY_APP (the default in UWP mode) will make headers hide Win32-only definitions. That's an easy way to find all the bad API calls.

I don't know if there's a definitive list somewhere... Opening the API docs under https://docs.microsoft.com/en-us/windows/win32/api shows on the bottom of the page if it's Desktop only, e.g. GetFileSizeEx:

![2020-03-17-10:24:57-screenshot](https://user-images.githubusercontent.com/5521887/76883600-a80bed80-6839-11ea-8d7b-2376b56e8df5.png)

Versus LoadLibraryA:

![2020-03-17-10:25:09-screenshot](https://user-images.githubusercontent.com/5521887/76883624-b1955580-6839-11ea-8556-c2f6b4d6bf1d.png)

Reply:
https://docs.microsoft.com/en-us/windows/win32/apiindex/umbrella-lib-onecoreuap